pkg/config/toml.go

```diff
@@ -61,6 +76,25 @@ func ReadFromToml(tomlPath string) (Toml, error) {
 	return t, nil
 }
 
+func ConvertToml(hostToml Toml, pwd string) Toml {
+	if len(hostToml.Meta.Dir) > 1 && hostToml.Meta.Dir[:1] == "~" {
+		hostToml.Meta.Dir = filepath.Join(pwd, hostToml.Meta.Dir[1:])
+	}
+	if len(hostToml.Data.StoreDataDir) > 1 && hostToml.Data.StoreDataDir[:1] == "~" {
+		hostToml.Data.StoreDataDir = filepath.Join(pwd, hostToml.Data.StoreDataDir[1:])
+	}
+	if len(hostToml.Data.StoreWalDir) > 1 && hostToml.Data.StoreWalDir[:1] == "~" {
+		hostToml.Data.StoreWalDir = filepath.Join(pwd, hostToml.Data.StoreWalDir[1:])
+	}
+	if len(hostToml.Data.StoreMetaDir) > 1 && hostToml.Data.StoreMetaDir[:1] == "~" {
+		hostToml.Data.StoreMetaDir = filepath.Join(pwd, hostToml.Data.StoreMetaDir[1:])
+	}
+	if len(hostToml.Logging.Path) > 1 && hostToml.Logging.Path[:1] == "~" {
+		hostToml.Logging.Path = filepath.Join(pwd, hostToml.Logging.Path[1:])
+	}
+	return hostToml
+}
+
 func GenNewToml(t Toml, path string) {
 	f, _ := os.Create(path)
```

Comment:
这里有问题。
1. err 需要处理
2. 建议这样写
```suggestion
	f, err := os.OpenFile(path, os.O_CREATE|os.O_TRUNC|os.O_RDWR, 0600)
```

Reply:
I have fixed it.